src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx

```diff
@@ -29,35 +36,40 @@ export const WalletMoreInfoCategory = ({
   const { t } = useTranslation("page-wallets-find-wallet")
 
   return (
-    <Box width="full" mt={4} _first={{ mt: 2 }}>
-      <Heading
-        as="h4"
-        lineHeight={1.4}
-        fontSize="md"
-        fontWeight={500}
-        mx="0.2rem"
-        mb={2}
-      >
+    <Box mx={{ base: 0, md: 0, lg: 2 }} w="100%">
+      {/* Category label */}
+      <Heading as="h4" lineHeight={1.4} fontSize="md" fontWeight="bold" mb={2}>
         {headingLabel}
       </Heading>
-      <Stack gap={2} wrap="wrap">
+
+      {/* Supported features */}
+      <UnorderedList m={0}>
         {orderedFeatureDropdownItems.map((feature) => {
           const featureColor = wallet[feature.filterKey!] ? "text" : "secondary"
+          const FeatureIcon = () => (
+            <Icon
+              as={
+                wallet[feature.filterKey!]
+                  ? GreenCheckProductGlyphIcon
+                  : WarningProductGlyphIcon
+              }
+              fontSize="md"
+              color={featureColor}
+            />
+          )
 
           if (feature.category === sectionName)
             return (
-              <HStack
+              <ListItem
                 key={feature.label}
-                spacing="0.2rem"
                 fontSize="0.9rem"
```

Comment:
Again not a blocker, but cc: @nloureiro if there is a way we can try to work our way towards standard font sizes